### ISSUE TITLE
Could not find 'AntDesign.interop.focus'

### ISSUE THREAD

**Issue by yorytang** [ant-design-blazor/ant-design-blazor#1117]
I'm using VS2019&NetCore5 version.

I have include ant-design-blazor components in my Blazor project, but when i using Select component it throw the error "Could not find 'AntDesign.interop.focus'"

**Comment by ElderJames** [comment]
Thanks for contacting us @yorytang .

Did you import the ant-design-blazor.js to your index.html file?
Or you can try to clear your browser cache.

**Comment by yorytang** [comment]
Hi, it's working now.

But there is another problem, the Select component can't using Search function,i have config the property 'EnableSearch', pls see my screenshot below, 

![image](https://user-images.githubusercontent.com/44489768/107173039-d2423d00-6a01-11eb-9880-5944645e7e79.png)
![image](https://user-images.githubusercontent.com/44489768/107173066-e5eda380-6a01-11eb-8d3e-e148b4366ffd.png)

**Comment by anddrzejb** [comment]
Can you post your code?

**Comment by yorytang** [comment]
pls see code below:

<AntDesign.Select Mode="tags"
                                      Placeholder="Please select"
                                      @bind-Values="@_selectedEmail"
                                      TItemValue="string"
                                      TItem="string"
                                      OnSelectedItemsChanged="OnSelectedItemsChangedHandler"
                                      EnableSearch>
                        <SelectOptions>
                            @foreach (var item in _items)
                                {
                                    <SelectOption TItemValue="string" TItem="string" Value=@item Label=@item />
                                }
                        </SelectOptions>
                    </AntDesign.Select>

**Comment by yorytang** [comment]
any update? thanks

**Comment by anddrzejb** [comment]
Sorry, I was crazy busy (to the point of working all night). I will try to look at this today. Will notify you about my findings.

**Comment by anddrzejb** [comment]
I tried with this code:
```razor
<AntDesign.Select Mode="tags"
    Placeholder="Please select"
    Style="width: 300px"
    @bind-Values="@_selectedEmail"
    TItemValue="string"
    TItem="string"
    OnSelectedItemsChanged="@OnSelectedItemsChangedHandler"
    EnableSearch>
    <SelectOptions>
    @foreach (var item in _items)
    {
        <SelectOption TItemValue="string" TItem="string" Label="@item.Item1" Value="@item.Item2"/>
    }
    </SelectOptions>
</AntDesign.Select>
Selected email: @string.Join(", ", _selectedEmail ?? Enumerable.Empty<string>())

@code {
    IEnumerable<string> _selectedEmail;
    List<(string, string)> _items = new List<(string, string)> {
        ("email1", "email1@dot.com"),
        ("email2", "email2@dot.com"),
        ("email3", "email3@dot.com"),
        ("email4", "email4@dot.com"),
    };

    private void OnSelectedItemsChangedHandler(IEnumerable<string> value)
    {
        if (value != null)
        Console.WriteLine($"Selected value: {string.Join(',', value)}");
    }
}
```
and everything is ok:
![issue1117](https://user-images.githubusercontent.com/6518006/107343823-6a0b5e00-6aca-11eb-8f12-344390eb090d.gif)

Maybe if you provided full snippet I would be able to help better?

**Comment by yorytang** [comment]
Hi，
i'm using the ABP blazor4.0 architecture, i have import ant blazor component in it.Is it a compatibility issue?
In addition to this issue, there is a Message component problem, i have inject MessageService and defined _message object ,but when running "await _message.Error("xxxx")" no any message Pop-up , no any error found in page.

**Comment by ElderJames** [comment]
Please add the <AntContainer /> in App.razor

**Comment by yorytang** [comment]
hi @ElderJames , i don't find the file in project you mentioned, do I need to create it ?

**Comment by anddrzejb** [comment]
I am not familiar with ABP so cannot comment directly, but every blazor application has an entry point. If you create a project from one of the default blazor templates, you will find file `App.razor`. In the basic template this is where routing is set up. Depending on which version you are using, it is wired differently:
* (server side) in `_Hosts.cshtml` there is a section similar to this:
```cshtml
<app>
   <component type="typeof(App)" param-InitialState="@Model.InitialState" render-mode="Server" />
</app>
```
where you can see `typeof(App)` - this directly refers to that mentioned `App.razor`
* (wasm) in `Program.cs` you have a line like this `builder.RootComponents.Add<App>("#app");` - this again refers to the same `App.razor` (also to div element that will be used for rendering all the html blazor generates).  

So you have to go through your template and find out which file is being used (from what you wrote, not `App.razor`, so which one?). Then check if inside that file you have `<AntContainer />`. Generally make sure that your template is following all the steps from [Import Ant Design Blazor into an existing project](https://github.com/ant-design-blazor/ant-design-blazor#import-ant-design-blazor-into-an-existing-project) section.

**Comment by yorytang** [comment]
@anddrzejb  I have create new blazor project(wasm& .net5) to testing the Select combo, but the problem still exists.
The step as below , please check if it is correct：

1. Import AntDesign components by Nuget
2. Add `builder.Services.AddAntDesign();` in `Program.cs`
3. Add `<link href="_content/AntDesign/css/ant-design-blazor.css" rel="stylesheet" />` and `<script src="_content/AntDesign/js/ant-design-blazor.js"></script>` in `index.xml`
4. Add `@using AntDesign` in `_Imports.razor`
5. Add `<AntContainer />` in `App.razor`

![image](https://user-images.githubusercontent.com/44489768/108299112-c7916000-71d8-11eb-950f-dcbaa077a4cb.png)

**Comment by anddrzejb** [comment]
Can you post that minimal repro? Zip, github, whatever.

**Comment by yorytang** [comment]
hi @anddrzejb 
please get demo from git
https://github.com/yorytang/ant-blazor-demo.git

**Comment by anddrzejb** [comment]
@yorytang I have it working, once I switched to latest nightly.
In your .csproj change antdesign to 
```xml
   <PackageReference Include="AntDesign" Version="0.7.0-nightly-2102141227" />
```

**Comment by yorytang** [comment]
@anddrzejb  thanks,it's working now. i think it was fixed in 0.7 version.
